apps/base-docs/docs/connecting-to-the-blockchain/configuring-providers-sbs.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Overview of Providers
+description: Configure several providers and use them to connect to the blockchain.
+hide_table_of_contents: false
+---
+
+[Rainbowkit] is a popular library that works with [wagmi] to make it easy to connect, disconnect, and change between multiple wallets. It's batteries-included out of the box, and allows for a great deal of customization of the list of wallets and connect/disconnect button.
```

Comment:
Similar comment around mentioning specific tooling from the Rainbow + wagmi team.